io.openems.edge.core/src/io/openems/edge/core/appmanager/dependency/AppManagerAppHelperImpl.java

```diff
@@ -1066,8 +1066,7 @@ private Optional<OpenemsAppInstance> findNeededApp(DependencyDeclaration declara
 		}
 		var appId = config.appId;
 		if (declaration.createPolicy == DependencyDeclaration.CreatePolicy.ALWAYS) {
-			var neededApps = this.getAllInstances().stream().filter(t -> t.appId.equals(appId))
-					.collect(Collectors.toList());
+			var neededApps = this.getAllInstances().stream().filter(t -> t.appId.equals(appId)).collect(Collectors.toList());
```

Comment:
you can also use `toList()` here or remove the change because its only formatting